src/utils/LogoMap.ts

```diff
@@ -0,0 +1,14 @@
+import AnywayImage from '../assets/anyway.png';
+import OrYarokLogo from '../assets/Or_Yarok.png';
+
+
+const orYarok = 'אור ירוק'
```

Comment:
use i18n

Reply:
actually, I  think that we need a drop-down for a fixed set of organizations.  your thoughts on this @yuvalbl ?

Reply:
This will be done on a different task.
Maybe it would be better to remove this until then